README.md

```diff
@@ -1,3 +1,7 @@
+#travis badge
```

Comment:
Why is this here?

Reply:
@DavidLemayian i have made changes on this.